src/Silk.Data/Migrations/20220704053342_ConfigMerge.cs

```diff
@@ -0,0 +1,339 @@
+                table: "infraction_steps",
+                newName: "IX_infraction_steps_GuildConfigEntityId");
+
+            migrationBuilder.RenameColumn(
+                name: "GuildModConfigEntityId",
+                table: "infraction_exemptions",
+                newName: "GuildConfigEntityId");
+
+            migrationBuilder.RenameIndex(
+                name: "IX_infraction_exemptions_GuildModConfigEntityId",
+                table: "infraction_exemptions",
+                newName: "IX_infraction_exemptions_GuildConfigEntityId");
+
+            migrationBuilder.AddColumn<int>(
+                name: "LoggingId",
+                table: "guild_configs",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<string>(
+                name: "NamedInfractionSteps",
+                table: "guild_configs",
+                type: "text",
+                nullable: false,
+                defaultValue: "");
+
+            migrationBuilder.AddColumn<bool>(
+                name: "ban_suspicious_usernames",
+                table: "guild_configs",
+                type: "boolean",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.AddColumn<bool>(
+                name: "delete_detected_phishing",
+                table: "guild_configs",
+                type: "boolean",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.AddColumn<bool>(
+                name: "detect_phishing",
+                table: "guild_configs",
+                type: "boolean",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.AddColumn<int>(
+                name: "max_role_mentions",
+                table: "guild_configs",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<int>(
+                name: "max_user_mentions",
+                table: "guild_configs",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<ulong>(
+                name: "mute_role",
+                table: "guild_configs",
+                type: "numeric(20,0)",
+                nullable: false,
+                defaultValue: 0ul);
+
+            migrationBuilder.AddColumn<bool>(
+                name: "progressive_infractions",
+                table: "guild_configs",
+                type: "boolean",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.AddColumn<bool>(
+                name: "use_native_mute",
+                table: "guild_configs",
+                type: "boolean",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.Sql
+            (
```

Comment:
Note: Could rename the `NamedInfractionSteps` column to match consistency of other columns, but this is very smol thing, can do after merge.